package.json

```diff
@@ -106,7 +108,7 @@
         "react-router-dom": "^5.2.0",
         "react-waypoint": "^9.0.3",
         "redux": "4.0.1",
-        "redux-logic": "^2.1.1",
+        "redux-logic": "^3.0.3",
```

Comment:
I updated this to try to fix something I think is a bug with redux -- it didn't fix it, but it didn't break anything else either, so I kept the update